nibabies/interfaces/images.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
+Image tools interfaces
+~~~~~~~~~~~~~~~~~~~~~~
+
+
+"""
+
+import nibabel as nb
+import numpy as np
+from nipype import logging
+from nipype.interfaces.base import (
+    BaseInterfaceInputSpec,
+    File,
+    SimpleInterface,
+    TraitedSpec,
+    traits,
+)
+from nipype.utils.filemanip import fname_presuffix
+
+LOGGER = logging.getLogger('nipype.interface')
+
+
+class RemapLabelsInputSpec(BaseInterfaceInputSpec):
```

Comment:
We actually already have a label remapping interface (https://github.com/nipreps/niworkflows/blob/a75d9c9fc9cc2e4e64da99ffc82a352d65d68ed1/niworkflows/interfaces/nibabel.py#L479-L507) that you should be able to use